app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/protectiveword/ProtectiveWordViewModel.kt

```diff
@@ -103,7 +103,7 @@ class ProtectiveWordViewModel @Inject constructor(
 }
 
 data class FetchMedicationUiState(
-    val onLoading: Boolean = false,
+    val onLoading: Boolean? = null,
```

Comment:
Why this change?

Reply:
used separate variable to control button enabling and disabling.